python/dgl/transforms/functional.py

```diff
@@ -3300,4 +3302,69 @@ def norm_by_dst(g, etype=None):
 
     return norm
 
+def rwpe(g, k):
```

Comment:
Can we support non-binary adjacency matrices for this case?

Reply:
added.